src/pages/docs/step-ca/provisioners.mdx

```diff
@@ -25,6 +26,20 @@ In this section we'll discuss the different provisioners, their target use cases
   - [K8sSA - Kubernetes Service Account](#k8ssa-kubernetes-service-account)
   - [Cloud Provisioners](#cloud-provisioners)
 
+## Choosing a Provisioner
+
+When choosing a provisioner for your use case,
+you can narrow your options by starting from the types of
+identifiers you need on a certificate. 
+
+Identifier Type | JWK | OAuth | X5C | ACME `dns-01` | ACME `http-01` | ACME `tls-alpn-01` | ACME `device-attest-01` | Nebula | SCEP | K8sSA | Cloud Provisioners
```

Comment:
Noticed `SSHPOP` is missing, but I think it's intentional, because of its somewhat limited use?

Reply:
Yeah, I intentionally left it out